docs/development/releasing.md

```diff
@@ -201,8 +251,34 @@ the correct checksums as specified in the SHASUMS files.
 
 ## Publish to npm
 
-Once the publish is successful, run `npm run publish-to-npm` to publish to
-release to npm.
+Before publishing to npm, you'll need to log into npm as Electron. Optionally,
+you may find [npmrc](https://www.npmjs.com/package/npmrc) to be a useful way
+to keep Electron's profile side-by-side with your own:
+```sh
+$ sudo npm install -g npmrc
+$ npmrc -c electron
+Removing old .npmrc (default)
+Activating .npmrc "electron"
+```
+
+The Electron account's credentials are stored in LastPass under the entry
+"Electron - NPM" for the URL "https://www.npmjs.com/login".
+```sh
+$ npm login
+Username: electron
+Password:
+Email: (this IS public) electron@github.com
+```
+
+Publish the release to npm.
+```sh
+$ npm whoami
+electron
+$ npm run publish-to-npm
+```
+
+Note: the `publish-to-npm` script may have trouble with Node 7 or higher. If
```

Comment:
Curious about this Node 7+ issue. What went wrong?

Reply:
@zeke when I have run publish-to-npm in the past it fails on newer versions of npm (eg npm 5).

Here is the error:
```
```node script/publish-to-npm.js
npm ERR! Cannot read property 'algorithm' of undefined

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jkleinsc/.npm/_logs/2017-08-30T20_50_53_319Z-debug.log
Error: Error: Command failed: npm publish /var/folders/yh/1n_9z9z97fn0tj_bpp3g9bqw0000gn/T/electron-npm117730-36976-1vxt06u.3cyk0dlsor/electron-1.7.7.tgz --tag beta
npm ERR! Cannot read property 'algorithm' of undefined

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jkleinsc/.npm/_logs/2017-08-30T20_50_53_319Z-debug.log```
```